api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml

```diff
@@ -839,8 +843,11 @@ components:
       type: object
     InstanceSelector:
       title: InstanceSelector
-      description: 'The name of an instance family (like "i3" or "m5d"),
-        or of a specific instance type (like "i3.large" or "m5d.xlarge") to target'
+      description: 'DEPRECATED use "instanceAllowList" instead'
+      type: string
+    InstanceAllowList:
+      title: InstanceSelector
+      description: 'A list of instance families (like "i3" or "m5d") or of a specific instance types (like "i3.large" or "m5d.xlarge") to target'
```

Comment:
I could be worth adding GCP examples too.

Reply:
Thanks! I did that